### PR TITLE
Tbkmini fix led

### PR DESCRIPTION
**Pull request by AdolfMuffin**
## Description

There was a problem with mapping a thumb-cluster leds, that I found on key-specific lightning effects, so I remapped the leds in config.
There wasn't VIA support "out of the box" in the default keymap - added.
There wasn't any glowing effect except static, so I added them into config.
Also, enabled NKRO as a feature.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bastardkb/tbkmini/keymaps/default/rules.mk
Comment (waffle87) [inline keyboards/bastardkb/tbkmini/keymaps/default/rules.mk:1]: This file should be remove. VIA is not to be enabled in the `default` keymap.

Comment (zvecr) [inline keyboards/bastardkb/tbkmini/keymaps/default/rules.mk:1]: This file needs to be removed. VIA cannot be enabled on the default keymap, and needs to be added to a `via` keymap like every other keyboard in the repo.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/bastardkb/tbkmini/v2/splinky_3/config.h [hunk 1]
```diff
@@ -29,3 +29,38 @@
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP17
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+#ifdef RGB_MATRIX_ENABLE
```
Comment (zvecr) [inline keyboards/bastardkb/tbkmini/v2/splinky_3/config.h:33]: These need to be enabled in info.json instead.

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by 0xcharly** [comment]
@AdolfMuffin thanks for the RGB matrix fix. 

If possible, I'd like to request to revert all other changes, since we're trying to keep the default keymaps simple and consistent across all BastardKB boards. It's easier for us to keep up with QMK and maintain those firmwares when the implementations are consistent

**Pull request closed without merging**